steps/nowsecure-severitycheck/0.0.1/step.yml

```diff
@@ -0,0 +1,43 @@
+title: |
+  NowSecure_SeverityCheck
+summary: |
+  Checking Severity of Critical/High/Medium issues in app bundle
+description: |
+  Step Checking the Critical/High/Medium Severity issues in app bundle and stop the build if any severity occurs.
+website: https://github.com/mhidaya90/bitrise-step-nowsecure-severitycheck
+source_code_url: https://github.com/mhidaya90/bitrise-step-nowsecure-severitycheck
+support_url: https://github.com/mhidaya90/bitrise-step-nowsecure-severitycheck
+published_at: 2023-01-09T13:18:43.711522855+05:30
+source:
+  git: https://github.com/mhidaya90/bitrise-step-nowsecure-severitycheck.git
+  commit: 850e0040dfc4310b1aeb04d0233ebd66017f1b0d
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+inputs:
+- nowsecure_api_token: null
+  opts:
+    description: |
+      NowSecure token generated from NowSecure Platform console
+    is_required: true
+    is_sensitive: true
+    summary: Your NowSecure Platform API token
+    title: NowSecure Platform API token
+- opts:
+    description: |
+      The platform associated with application. Allowed values- "ios", "android"
+    is_required: true
+    is_sensitive: true
+    summary: The platform associated with application. Allowed values- "ios", "android"
+    title: Application Platform
+  platform: null
```

Comment:
Since there are specific values please utilize the `value_options` field. Here's an example from the [flutter-build](https://github.com/bitrise-steplib/bitrise-step-flutter-build/blob/master/step.yml#L72) step:

```
inputs:
- platform: both
  opts:
    title: Platform
    summary: The selected platform will be built, or both iOS and Android if you select both
    description: The selected platform will be built, or both iOs and Android if you select both
    is_required: true
    value_options:
    - both
    - ios
    - android
    ```